modules/openshift-developer-cli-basic-odo-cli-commands.adoc

```diff
@@ -395,14 +419,19 @@ Perform storage operations.
 ----
   # Create storage of size 1Gb to a component
   odo storage create mystorage --path=/opt/app-root/src/storage/ --size=1Gi
+
   # Delete storage mystorage from the currently active component
   odo storage delete mystorage
   
   # Delete storage mystorage from component 'mongodb'
   odo storage delete mystorage --component mongodb
+  
   # List all storage attached or mounted to the current component and
   # all unattached or unmounted storage in the current application
   odo storage list
+  
+  # Sse the` -o json` flag to get a JSON formatted output
```

Comment:
* Sse -> Set
* Fix the backtick so it's properly around "-o json"

Reply:
@bergerhoffer fixed! Thank you!